### PR TITLE
starting on rating diagram

### PR DESCRIPTION
**Pull request by jordansread**

### CODE REVIEW

File: R/uvhydrograph-fnc.R [hunk 1]
```diff
@@ -164,70 +164,42 @@ combineLims<- function(lims1, lims2, ...) {
...
   day1 <- xticks[strftime(xticks, format = '%d') == "01"]
   if (ylog){
-    yticks <- .closestLogged(10^pretty(par()$usr[3:4], num_maj_y))
-    yminor <- .betweenLogs(10^par()$usr[3:4])
+    yticks <- .closestLogged(10^pretty(lims$ylim, num_maj_y))
+    yminor <- .betweenLogs(lims$ylim)
   } else {
-    yticks <- pretty(par()$usr[3:4], num_maj_y)
-    yminor <- pretty(par()$usr[3:4], num_min_y)
+    yticks <- pretty(lims$ylim, num_maj_y)
```
Comment (jordansread) [inline R/uvhydrograph-fnc.R:207]: @thongsav-usgs would like to do this with `logTicks()` if you are ok with that.

**Comment by jordansread** [comment]
@thongsav-usgs I am stopping here. This is what the html version looks like:
image
Will pause until sprint, unless something else changes

**Comment by jordansread** [comment]
the shifts are fake, as we are waiting for that in JSON

**Comment by thongsav-usgs** [comment]
Perfect, that's more than enough to at least demo the progress the Mike had made.

**Pull request merged**